src/utils/components/AddBootableVolumeModal/components/VolumeDestination/VolumeDestination.tsx

```diff
@@ -24,6 +28,13 @@ const VolumeDestination: FC<VolumeDestinationProps> = ({
 
   const { bootableVolumeName, size, storageClassName } = bootableVolume || {};
 
+  const [applyStorageProfile, setApplyStorageProfile] = applyStorageProfileState;
+
+  const { claimPropertySets, loaded: storageProfileLoaded } =
+    useStorageProfileClaimPropertySets(storageClassName);
```

Comment:
This hook is also used in the parent, can you get what you need as a prop and save a network call?

Reply:
Yes. That was an oversight. Thanks for catching it.